### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by MIYankov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+### Styling
+This template is set up to use SASS for styling. All classes used are based on the [{N} Theme](https://docs.nativescript.org/angular/ui/theme.html#theme). Check it out to see what classes you can use on which component.
+
+It has 4 global style files that are located at the root of the app folder:
+- `_app-variables.scss` - holds the global SASS variables that are imported on each component's styles.
+- `app.scss` - the global common style sheet. These style rules will be applied to both Android and iOS.
+- `platform.android.scss` - the global Android style sheet. These style rules will be applied to Android only.
+- `platform.ios.scss` - the global iOS style sheet. These style rules will be applied to iOS only.
+
+Each component including the drawer custom component has 3 style files located in the folder of the component:
```
Comment (vegeorgi) [inline README.md]: Has we mention "drawer" here, as in this template there is no drawer component?

**Comment by manoldonev** [comment]
ping @estoychev

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
-# template-tab-navigation-ng
+# NativeScript Angular Tab Navigation Template
```
Comment (estoychev) [inline README.md]: NativeScript with Angular Tab Navigation App Template

File: README.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+
+## Intro - TODO
+
```
Comment (estoychev) [inline README.md]: App template featuring a TabView component for navigation.

**Pull request merged**